Group entities that belong to the same device

tetienne opened the pull request:
Some of my device urls look like: "io://xxxxx/4468654#10", "io://xxxxx/4468654#9"... 
We can use this to group several entities in the same device.

vlebourl commented on custom_components/tahoma/tahoma_device.py:
```diff
@@ -119,7 +119,7 @@ def device_state_attributes(self):
+            "identifiers": {(DOMAIN, self.unique_id.split("#", 1)[0])},
             "manufacturer": "Somfy",
             "name": self.name,
             "model": self.tahoma_device.widget,
```
name and model should not stay I guess, at least in the Thermostat, they're not the same for all 3 entities.

tetienne commented:
>>> unique_id="io://xxxxx/4468654#10"
>>> unique_id.split("#", 1)[0]
'io://xxxxx/4468654'
>>> unique_id = "foo"
>>> unique_id.split("#", 1)[0]
'foo'

vlebourl commented:
This is rejected as not possible in a satisfactory as Somfy doesn't have the Device abstraction as HA does.
